src/coreclr/src/interop/comwrappers.cpp

```diff
@@ -179,16 +180,21 @@ namespace ABI
     }
 }
 
+// ManagedObjectWrapper_QueryInterface needs to be visible outside of this compilation unit
+// to support the DAC. See code:ClrDataAccess::DACTryGetComWrappersObjectFromCCW for the
+// usage in the DAC (look for the GetEEFuncEntryPoint call).
+
+HRESULT STDMETHODCALLTYPE ManagedObjectWrapper_QueryInterface(
```

Comment:
Could we instead put `ABI::ToManagedObjectWrapper(...)` in the new ABI header?

Reply:
We would have to DACize the implementation of MapFromIUnknown/ToManagedObjectWrapper. And even then we'd likely have to still keep some of the custom DAC support (the equivalent "map IUnknown to CCW" code path for the built-in CCWs is duplicated in the DAC).

Reply:
Oh because that function is actually running in the debuggee process not in the debugger process right?

Reply:
Yep